Can't build loqui anymore

I build Loqui frequently to check if there's any improvements, but I tried to do it today and it failed at "npm install". The log file is really long, so here's a link to it on gist: https://gist.github.com/FabianOvrWrt/b5a330c597853df58cad#file-npm-debug-log

I don't know if anything changed or if I'm doing something wrong.

I as well have problems with building loqui but on windows, and I think it is not the same problem as yours. Maybe someone could help me. This is the output from the cmd:

```
npm WARN package.json LoquiIM@v0.4.6 license should be a valid SPDX license expr
ession

> contextify@0.1.14 install D:\ZTE\Spielereien\loqui\im\node_modules\jsdom\node_
modules\contextify
> node-gyp rebuild

/
D:\ZTE\Spielereien\loqui\im\node_modules\jsdom\node_modules\contextify>if not de
fined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\no
de-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node
rebuild )
Die Projekte in dieser Projektmappe werden nacheinander erstellt. Um eine parall
ele Erstellung zu ermöglichen, müssen Sie den Schalter "/m" hinzufügen.
c:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\src\win_delay_lo
ad_hook.c(13): fatal error C1083: Datei (Include) kann nicht geöffnet werden: "
windows.h": No such file or directory [D:\ZTE\Spielereien\loqui\im\node_modules
\jsdom\node_modules\contextify\build\binding.sln]
build error
gyp ERR! stack Error: `C:\Windows\Microsoft.NET\Framework\v4.0.30319\msbuild.exe
` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\
npm\node_modules\node-gyp\lib\build.js:270:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_proces
s.js:200:12)
gyp ERR! System Windows_NT 6.1.7601
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodej
s\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd D:\ZTE\Spielereien\loqui\im\node_modules\jsdom\node_modules\context
ify
gyp ERR! node -v v4.1.0
gyp ERR! node-gyp -v v3.0.3
gyp ERR! not ok
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\
node_modules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v4.1.0
npm ERR! npm  v2.14.3
npm ERR! code ELIFECYCLE

npm ERR! contextify@0.1.14 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the contextify@0.1.14 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the contextify package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls contextify
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\ZTE\Spielereien\loqui\im\npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -19,14 +19,14 @@
   "pre-commit": ["jshint"],
   "devDependencies": {
     "grunt": "~0.4.1",
-    "grunt-contrib-clean": "~0.5.0",
-    "grunt-contrib-compress": "~0.9.1",
-    "grunt-contrib-connect": "~0.6.0",
-    "grunt-contrib-copy": "~0.5.0",
-    "grunt-contrib-watch": "~0.5.3",
+    "grunt-contrib-clean": "~0.6.0",
+    "grunt-contrib-compress": "~0.13.0",
+    "grunt-contrib-connect": "~0.11.2",
+    "grunt-contrib-copy": "~0.8.1",
+    "grunt-contrib-watch": "~0.6.1",
     "grunt-execute": "~0.2.2",
-    "grunt-node-webkit-builder": "~0.2.0",
-    "jsdom": "~2.0.0",
+    "grunt-node-webkit-builder": "~1.0.2",
+    "jsdom": "~6.5.1",
     "jshint": "^2.6.0",
     "pre-commit": "^1.0.1"
   }

